pypeit/par/pypeitpar.py

```diff
@@ -1388,6 +1388,29 @@ def __init__(self, slit_spec=None, relative_weights=None, combine=None, output_f
                                    'ranges. If combine=False, the individual spec3d files will have a suffix ' \
                                    '"_whitelight".'
 
+        defaults['method'] = "subsample"
+        dtypes['method'] = str
+        descr['method'] = 'What method should be used to generate the datacube. There are currently two options:' \
+                          '(1) "subsample" (default) - this algorithm subsamples each pixel in the spec2d frames ' \
+                          'and assigns each subpixel into a voxel of the datacube. Flux is conserved, but voxels ' \
+                          'are correlated, and the error spectrum does not account for covariance between adjacent ' \
+                          'voxels. ' \
+                          '(2) "NGP" (nearest grid point) - this algorithm is effectively a 3D histogram. Flux is ' \
```

Comment:
Nice docs. I suggest adding what you wrote here to the core routine to address my comment above.

Reply:
Maybe clarify that when multiple pixels hit the same bin for NGP, you are taking the average flux. I think that SB and hence flux are still conserved, but maybe worth a note since your docs suggest it would just add the flux.